docs/Documentation/Applications/gams.md

```diff
@@ -0,0 +1,101 @@
+# Using the General Algebraic Modeling System
+
+*The General Algebraic Modeling System (GAMS) is a commercial high-level modeling system for mathematical programming and optimization. It is licensed software.*
+
+GAMS includes a DSL compiler and also a stable of integrated high-performance solvers. GAMS is able to solve complex, large-scale modeling problems. For documentation, forums, and FAQs, see the [GAMS website](https://www.gams.com/).
+
+A 40-user license of GAMS is made available to Eagle users. The GAMS license requires users to be a member of a "workgroup." If you need the GAMS software or a specific solver from GAMS, or if you have trouble running GAMS, please [contact us](https://www.nrel.gov/hpc/contact-us.html).
+
+## Initializing Your Environment
+
+To initialize your environment to use GAMS, simply type `module load gams/<version>` — see `module avail gams` output to see available versions. GAMS is run with the command format `gams <input filename>`. A file `<input filename>.lst` will be created as the output file.   
+
+For a test run, in your home directory, type the following:
+
+```
+module load gams/<version>
+cp /nopt/nrel/apps/gams/example/trnsport.gms .
+gams trnsport
+```
+
+A result of 153.675 should be found from screen output. More detailed output is in the file `trnsport.lst`. 
+
+## Selecting an Alternative Solver
+
+The available solvers for different procedures are shown in the following with the default solver being the first one:
+
+* LP: **GUROBI** BDMLP CBC IPOPT SOPLEX
+* MIP: **GUROBI** BDMP CBC SCIP
+* RMIP: **GUROBI** BDMLP CBC IPOPT SOPLEX
+* NLP: **SCIP** COUENNE IPOPT
+* MCP: **NLPEC** MILES
+* MPEC: **NLPEC**
+* RMPEC: **NLPEC**
+* CNS: **SCIP** COUENNE IPOPT 
+* DNLP: **SCIP** COUENNE IPOPT 
+* RMINLP: **SCIP** COUENNE IPOPT 
+* MINLP: **SCIP** BONMIN COUENNE 
+* QCP: **GUROBI** COUENNE IPOPT SCIP
+* MIQCP: **GUROBI** BONMIN COUENNE SCIP
+* RMIQCP: **GUROBI** COUENNE IPOPT SCIP
+* EMP: **JAMS** LOGMIP SELKIE 
+
+By typing `gams <input_filename>` on the command line, the default procedure LP and the default solver Gurobi will be used. In order to override the default option to use, *e.g.*, Soplex, you can try the following two methods: 
+
+[1]  Use the *option* statement in your GAMS input file. For example, if your model input uses LP procedure and you want to use Gurobi solver to solve it, just add `option lp=soplex` to your input file.
+
+or
+
+[2]  Specify the solver in the command line, *e.g.*, `gams <input_filename> lp=soplex`. 
+
+A sample script for batch submission is provided here:
+
+??? example "Eagle Sample Submission Script"
+
+	```
```

Comment:
Can add code highlighting by adding language name. So for this script, should be ``` bash